src/screens/CreateEventPage.js

```diff
@@ -10,15 +11,28 @@ class CreateEventPage extends Component {
       this.state = {
          fields: {},
          errors: {},
-         formIsValid: true
-
+         formIsValid: true,
+         showModal:false,
+         isShowing: false
```

Comment:
We can probably combine these into one state object such as `shouldShowModal` if they aren't logged in then it will be true and if they are logged in then it will be false. Then we can use the `shouldShowModal` state to decide if we show the modal popup